addons/udes_stock/tests/test_stock_picking.py

```diff
@@ -2005,10 +2007,30 @@ def test_picking_cannot_be_deleted_outside_draft(self):
         with self.assertRaises(UserError):
             pick.unlink()
 
+    def test_picking_can_be_deleted_with_support_users(self):
+        """
+        Test that a picking can be deleted with support users
+        """
+        self.env.user.groups_id += self.env.ref("udes_security.group_trusted_user")
+        pick = self.create_picking(
+            picking_type=self.picking_type_pick,
+            products_info=[{"product": self.fig, "uom_qty": 10}],
+            location_id=self.test_stock_location_02.id,
+            confirm=False,
+            assign=False,
+            priority="0",
+        )
+        pick.action_cancel()
+        self.assertEqual(pick.state, "cancel")
+        pick.unlink()
+        self.assertFalse(pick.exists())
+
     def test_picking_can_be_deleted_with_bypass_state_check(self):
         """
         Test that a picking can be deleted with context variable
         """
+        # Making sure is not a support user
```

Comment:
nit, we interchange `support` and `trusted`. maybe we should just stick to `trusted` as that's the group name

Reply:
Pushed the suggested change, thanks